linkerd/proxy/identity/src/certify.rs

```diff
@@ -35,9 +35,9 @@ pub struct Config {
 #[pin_project]
 #[derive(Clone, Debug)]
 pub struct Local {
```

Comment:
nit/TIOLI: might be worth renaming this now that we have a type called `LocalId`, to destinguish it a little more clearly. Maybe something like `LocalCert`?